devices.yaml

```diff
@@ -15,11 +15,74 @@ glider_devices:
     make: Wetlabs
     model: FLNTU
     serial: '6661'
+    long_name: Wetlabs ECO Puck FLNTU
+    make_model: Wetlabs FLNTU
+    factory_calibrated: "Yes"
+    calibration_date: "2021-02-01"
+    calibration_parameters: Chl_DarkCounts = 48 Chl_SF = 0.0121 NTU_DarkCounts = 49 NTU_SF = 0.0060
```

Comment:
Let's make these calibration parameters easy to read for downstream users. How about formatting as a dictionary?:

{Chl_DarkCounts : 48,  Chl_SF : 0.0121, NTU_DarkCounts : 49, NTU_SF  0.0060}